test/Erlang/ConfigParserSpec.hs

```diff
@@ -104,19 +101,20 @@ spec = do
 
     it "should parse everything at once" $
       parse parseConfig "stresstest.config" oneWithEverything `shouldParse` 
-        [ErlTuple [
-          atom "rawAtom",
-          atom "quotedAtom",
-          ErlString "Regular String",
-          ErlString "Escaped \" String",
-          ErlInt 1234, -- Literal
-          ErlFloat 3.14159,
-          ErlInt 120, -- '$x'
-          ErlInt 35338, -- 'wes' in base 33
-          ErlArray [atom "arr1"],
-          ErlArray [ErlTuple [atom "key", ErlString "value"]],
-          ErlTuple [atom "number", ErlInt 5678] -- Literal
-        ]]
+        ConfigValues 
+          [ErlTuple [
```

Comment:
[nit] Would it make sense to have multiple `ErlTuple` values in the `ConfigValues` array for the test now that we made that possible with the `ConfigValues` type?

Reply:
Not sure I understand.  It already supports multiple `ErlValues`.

Reply:
Yea that makes sense, I was asking if we should modify the test values to contain multiple `ErlValues` so that we can test it

Reply:
Ah, the real-life test already checks that.  It fails if the parser doesn't sit just right.  [Check here](https://github.com/fossas/spectrometer/pull/139/files#diff-502443b22c0ddb7b11e6fe0f961f459cR24)